kedro/extras/datasets/dask/parquet_dataset.py

```diff
@@ -6,6 +6,7 @@
 
 import dask.dataframe as dd
 import fsspec
+import triad
```

Comment:
Would it be possible to import `triad` only if the user has provided a schema argument? It would be nice if the library wasn't mandatory if you're not using the schema. But no big deal if that makes things too complicated!

Reply:
I tried moving the import inside the `_process_schema` method, but I ended up getting linting errors from the pre-commit suite. 😢

```
************* Module kedro.extras.datasets.dask.parquet_dataset
kedro/extras/datasets/dask/parquet_dataset.py:178:8: C0415: Import outside toplevel (triad) (import-outside-toplevel)
```

Reply:
No worries. Let's leave it as is. I realised it might be confusing for a user if they've already installed the dataset, but then when they add `schema` they need to install another dependency.